Index: swiginc/image.i
===================================================================
RCS file: /data2/cvsroot/mapserver/mapscript/swiginc/image.i,v
retrieving revision 1.12
diff -c -r1.12 image.i
*** swiginc/image.i     18 Feb 2005 20:08:01 -0000      1.12
--- swiginc/image.i     27 Nov 2005 06:52:08 -0000
***************
*** 117,122 ****
--- 117,135 ----
      }
  #endif
  
+ #if defined SWIGTCL8
+ %{
+ #include <tclgd.h>
+ %}
+ 
+     int gdHandle (char *newCommandName)
+     {
+       return tclgd_newGDObjectAttach (SWIG_TCL_INTERP, 
+                                       newCommandName, 
+                                       self->img.gd);
+     }
+ 
+ #endif
      /* -----------------------------------------------------------------------
         saveToString is deprecated and no longer supported.  Users should use
         the write() method instead.
